### DISCUSSION TITLE
Weird Predictions when using own Exogenous variables for time series model

### DISCUSSION THREAD

**Discussion by thommyeur** [pycaret/pycaret#3547] [Q&A]
Hi guys,

I'm a bit stuck on this problem I am facing with my time series forecasting model.

I am predicting the future production quantities ten days into the future based on exogenous variables like outside temperature and yearweekday. To retrieve the values for the outside temperature I make use of the actual outside temperature forecast but when I use these values the eventual model predictions range from -4 to 4 whilst a range of 25K to 50K is expected.

The weird thing is that when I use another time series model to predict the values for the exogenous variables the predictions are what I expect them to be. I've used the way to predict exogenous variables from the tutorial on the Pycaret website.

Did anyone encounter this before and can help me solve this? 

Much thanks in advance :)

**Comment by ngupta23** [comment]
Not faced this before. Please share a reproducible example so we can debug what is going on.

**Reply by thommyeur** [reply]
Hi Nikhil, 

Thanks for your reply. How should I deliver it, with example data and the code I've used?

**Reply by ngupta23** [reply]
Please file an issue. That will point you through the process. If you can share a colab notebook with the code and data, it would be ideal.
